### PR TITLE
LPD-22173 fix poshi test

### PR DESCRIPTION
**Pull request by algobob**

### CODE REVIEW

**Comment by algobob** [comment]
ci:test:relevant

**Comment by algobob** [comment]
ci:help

**Comment by algobob** [comment]
ci:test:poshi

**Comment by algobob** [comment]
ci:test:poshi

File: modules/apps/portal-language/portal-language-override-test/src/testFunctional/macros/LanguageOverride.macro [hunk 1]
```diff
@@ -94,6 +94,7 @@ definition {
 	@summary = "Default summary"
 	macro exportOverriddenTranslations() {
 		Portlet.gotoPortletOptions(portletOption = "Export Overridden Translations");
+		DownloadTempFile();
```
[the remark this answers is not in the document]
Comment (algobob) [reply modules/apps/portal-language/portal-language-override-test/src/testFunctional/macros/LanguageOverride.macro:98]: done

**Comment by ethib137** [comment]
ci:forward

**Pull request closed without merging**